manifests/plugin/remote_execution/ssh.pp

```diff
@@ -0,0 +1,67 @@
+# = Foreman Proxy Remote Execution SSH plugin
```

Comment:
See https://github.com/ekohl/puppet-foreman_proxy/commit/d75c6819c20360397fd3cf8fb4e62ecfa3074469 for an example.